src/components/timePicker/timePicker.tsx

```diff
@@ -0,0 +1,48 @@
+import {useForm } from "react-hook-form";
+
+type TimePickerProps = {
+  label?: string;
+}
+const TimePicker = ({ label }: TimePickerProps) => {
+  const { register, formState: { errors } } = useForm();
+  return (
+    <div className="w-346 h-99">
+    <label className="text-base leading-[19.2px] font-rubik mb-4">{label}</label>
```

Comment:
I wouldnt do ```19.2px``` I would just do ```19px``` this is usually a designer error when they have a weird decimal number.

Reply:
Fixed that.